src/Common/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -6,12 +6,14 @@ namespace System.Numerics.Hashing
 {
     internal static class HashHelpers
     {
+        public static int RandomSeed { get; } = Guid.NewGuid().GetHashCode();
```

Comment:
Related issue https://github.com/dotnet/coreclr/issues/2279

Could be solved by wrapping key in a ValueTuple? 

/cc @jskeet